nat-lab/tests/uniffi/serialization.py

```diff
@@ -0,0 +1,62 @@
+import inspect
+import Pyro5.api  # type: ignore
+
+
+def init_serialization(libtelio):
+    all_members = inspect.getmembers(libtelio, inspect.isclass)
+    public_members = []
+    for name, cls in all_members:
+        if not name.startswith("_"):
+            public_members.append(cls)
+            nested_members = inspect.getmembers(cls, inspect.isclass)
+            for name, cls in nested_members:
+                if not name.startswith("_"):
+                    public_members.append(cls)
+
+    classes = tuple(
+        filter(
+            lambda m: str(m).startswith("<class ") and libtelio.__name__ in str(m),
+            public_members,
+        )
+    )
+    enums = tuple(filter(lambda m: str(m).startswith("<enum "), public_members))
```

Comment:
For filtering out enums and classes, did you explore inspect.issubclass ? Just wondering why a string based filtering instead instead of a similar approach as above.

Reply:
Idk if it will work, but can you try it out with `issubclass` and `not issubclass` and see ? If it works thats great, if not then keep it as it is!